### PR TITLE
320 booking info component

### LINKED ISSUE #320: [FRONTEND] Create booking information component
**Is your feature request related to a problem? Please describe.**

This should be in a folder called `Bookings` alongside the other `composite` components. We should only care about presentation in this component, so expose props allowing us to adjust the listed prices.

Image

**BEFORE MERGING**

- [ ] Storybooks created where possible
- [ ] Tested with [nock](https://tanstack.com/query/v3/docs/framework/react/guides/testing) if data fetching
- [ ] Tests written for critical interactions
- [ ] PR Reviewed (For non-trivial changes)
- [ ] Changes tested after rebasing on master or merging in master (_hint_: `git fetch origin master:master`, then `git rebase master` or `git merge master`)
- [ ] All required PR checks passing

### PR DESCRIPTION
**Pull request by zlrkw11**
closes #320

### CODE REVIEW

**Comment by zlrkw11** [comment]
image

**Review by bcho892 — CHANGES_REQUESTED** [review]
image

^ from the ticket

We shouldn't hard code the prices as they may change

**Comment by zlrkw11** [comment]
image
resolved the issue.

File: client/src/components/composite/Booking/BookingInfoComponent/BookingInfoComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+type props = IBookingInfoProps
+
+const Divider = () => (
+  <span className="bg-dark-blue-100 m-8 h-[1px] w-[283px]" />
+)
+
+const BookingInfoComponent = ({ price1, price2, price3 }: props) => {
+  return (
+    <div className="border-gray-3 flex h-[474px] max-w-[347px] flex-col items-center rounded border">
```
Comment (bcho892) [inline client/src/components/composite/Booking/BookingInfoComponent/BookingInfoComponent.tsx:15]: this should be full width with content being centered on smaller screen so its responsive. height should be set to fit content with appropriate paddings

File: client/src/components/composite/Booking/BookingInfoComponent/BookingInfoComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+interface IBookingInfoProps {
+  price1: string
```
Comment (bcho892) [inline client/src/components/composite/Booking/BookingInfoComponent/BookingInfoComponent.tsx:2]: nit: these names aren't particularly descriptive

File: client/src/components/composite/Booking/BookingInfoComponent/BookingInfoComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  price2: string
+  price3: string
+}
+
+type props = IBookingInfoProps
+
+const Divider = () => (
+  <span className="bg-dark-blue-100 m-8 h-[1px] w-[283px]" />
```
Comment (bcho892) [inline client/src/components/composite/Booking/BookingInfoComponent/BookingInfoComponent.tsx:10]: again, not recommended to use fixed widths

File: client/src/components/composite/Booking/BookingInfoComponent/BookingInfoComponent.tsx (hunk 2 above)
Comment (zlrkw11) [reply client/src/components/composite/Booking/BookingInfoComponent/BookingInfoComponent.tsx:2]: done

**Comment by zlrkw11** [comment]
image
- new small viewport 

image
- largest desktop viewport

image
- large mobile viewport

image
- tablet viewport

**Review by bcho892 — APPROVED** [review]

**Pull request merged**